src/main/java/it/gov/pagopa/reporting/RetrieveFlows.java

```diff
@@ -99,6 +101,8 @@ public void run(
                             } else {
                                 logger.log(Level.SEVERE, () -> "[NODO Connection down]  Max retry exceeded.");
                             }
+                        } catch (AppException e) {
+                            logger.log(Level.SEVERE, () -> "[CACHE] Organization: [" + organization +"] Caused by: " + e.getCause() + " Message: " + e.getMessage());
```

Comment:
I prefer remove `[CACHE]` from the log. In the future the `AppException` is generic so could be tied to or generated at another point in the flow